documentation/source/user_section/tutorials/4d/processing-fmri-data/preprocessing.rst

```diff
@@ -0,0 +1,83 @@
+-------------------------------------
+
+Due to the low contrast between spinal cord and cerebrospinal fluid of fMRI data, it is difficult to directly obtain a spinal cord segmentation for fMRI data using ``sct_deepseg_sc``. So, as a workaround, we will instead obtain a spinal cord segmentation for another contrast.
+
+.. code::
+
+   cd ../t2
+   sct_deepseg_sc -i t2.nii.gz -c t2
+
+:Input arguments:
+   - ``-i`` : Input image
+   - ``-c`` : Contrast of the input image
+
+:Output files/folders:
+   - ``t2_seg.nii.gz`` : 3D binary mask of the segmented spinal cord
+
+
+Now that we have the spinal cord segmentation for the T2 image, we can transform that segmentation into the space of the fMRI data by registering the T2 segmentation with the fMRI mean image created earlier.
+
+.. code::
+
+   cd ../fmri
+   sct_register_multimodal -i ../t2/t2_seg.nii.gz -d fmri_mean.nii.gz -identity 1
+
+
+:Input arguments:
+   - ``-i`` : Input image
+   - ``-d`` : Destination image
+   - ``-identity`` : Just put source into destination (no optimization). (TODO: This description is a little vague/confusing for me. How does this mean ``-identity``? If no optimiation is performed, then what does this accomplish? Can we clarify this in the help description of ``sct_register_multimodal``.)
+
+:Output files/folders:
+   - ``t2_seg_reg.nii.gz`` : The T2 segmentation, transformed to the space of the fMRI mean image. This file is what will be used to create the spinal cord mask for fMRI motion correction.
+   - ``fmri_mean_reg.nii.gz`` : The fMRI mean image, transformed to the space of the T2 segmentation.
+   - ``warp_t2_seg2fmri_mean.nii.gz`` : The warping field to transform the T2 segmentation to the fMRI space.
+   - ``warp_fmri_mean2t2_seg.nii.gz`` : The warping field to transform the fMRI mean image to the T2 space.
+
+Creating a mask around the spinal cord
+--------------------------------------
+
+Now that we have a spinal cord segmentation in the space of the fMRI data, we can use it to create a mask around the fMRI spinal cord.
+
+.. code::
+
+   sct_create_mask -i fmri.nii.gz -p centerline,t2_seg_reg.nii.gz -size 35mm -f cylinder
+
+:Input arguments:
+   - ``-i`` : The input image to create the mask from.
+   - ``-p`` : The process used to position the mask. The ``centerline`` process will compute the center of mass for each slice of ``t2_seg_reg.nii.gz``, then use those locations to center of the mask at each slice.
+   - ``-f``: The shape of the mask. Here, we create cylinder around the centerline.
+   - ``-size``: The {diameter? radius? TODO: Clarify in help description.} of the mask.
+
+:Output files/folders:
+   - ``mask_fmri.nii.gz`` : An imagine containing a mask surrounding the spinal cord.
+
+.. TODO: Why is it that we use the mask crop the image for dMRI MOCO, but for fMRI MOCO, we pass the mask directly to the function? Shouldn't these two tutorials be consistent? Why don't we use `sct_dmri_moco -m` for the dMRI tutorial?
```

Comment:
I notice that the dMRI and fMRI tutorials have slightly different uses of the mask. I'm wondering if we might want to either:

1. Pass the mask directly to `sct_dmri_moco`
2. Crop the fMRI data so the steps match
3. Explain the rationale behind having different steps.